api/tests/routes/pro/get_offerer_addresses_test.py

```diff
@@ -163,9 +165,13 @@ def test_get_offerer_addresses_is_editable(self, client, linked_to_venue):
         )
 
         if linked_to_venue:
-            offerers_factories.VenueFactory(managingOfferer=offerer, offererAddress=offerer_address_1)
+            venue = offerers_factories.VenueFactory(managingOfferer=offerer, offererAddress=offerer_address_1)
         client = client.with_session_auth(email=pro.email)
         offerer_id = offerer.id
+        offerer_address_1_id = offerer_address_1.id
+        offerer_address_2_id = offerer_address_2.id
+        offerer_address_3_id = offerer_address_3.id
+        venue_public_name = venue.common_name if linked_to_venue else None
```

Comment:
```suggestion
        venue_public_name = venue.common_name if linked_to_venue else "1ere adresse"
```
et supprimer la condition dans `"label": venue_public_name or "1ere adresse",`
Point bonus, renommer le test (`test_get_offerer_addresses_is_editable`) vu que l'on n'a plus de champs `isEditable` 🙏